From 387771175dbf0076df8186f243c7ef401e3da0a4 Mon Sep 17 00:00:00 2001
From: micky387 <mickaelsaibi@free.fr>
Date: Wed, 22 Nov 2017 09:31:00 +0100
Subject: [PATCH 2/2] qca1530: fix neverallow on adbd

(neverallow adbd base_typeattr_154 (process (transition)))
  <root>
  allow at /home/micky387/Lineage-15/out/target/product/tbltexx/obj/ETC/nonplat_sepolicy.cil_intermediates/nonplat_sepolicy.cil:11650
    (allow adbd_26_0 qca1530 (process (transition)))

Change-Id: I367086c1744bcd13f43322218a2b7a6d721e53bf
---
 apq8084/qca1530.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/apq8084/qca1530.te b/apq8084/qca1530.te
index f1cd348..328028a 100644
--- a/apq8084/qca1530.te
+++ b/apq8084/qca1530.te
@@ -32,7 +32,6 @@ init_daemon_domain(qca1530)
 
 userdebug_or_eng(`
   domain_auto_trans(shell, qca1530_exec, qca1530)
-  domain_auto_trans(adbd, qca1530_exec, qca1530)
 ')
 
 qmux_socket(qca1530)
-- 
2.7.4

